packages/sucrase/src/index.ts

```diff
@@ -28,7 +30,7 @@ module.exports = function sucrase(opts = {}) {
       }
     },
 
-    transform(code, id) {
+    transform(code: any, id: any) {
```

Comment:
if we're going to type this, let's not use `any`. If it's too heavy a lift to type properly, please revert.